hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -282,6 +290,15 @@ public void addRpcProcessingTime(long processingTime) {
     }
   }
 
+  public void addRpcResponseTime(long processingTime) {
```

Comment:
change argument name to ``responseTime``

Reply:
Having the unit would help

Reply:
changed. Thank you

Reply:
@LiuGuH can you address the unit part as well?